locale/en/LC_MESSAGES/djangojs.po

```diff
@@ -4449,6 +4448,10 @@ msgstr ""
 msgid "Are you sure you want to delete this request?"
 msgstr ""
 
+#: corehq/apps/users/static/users/js/web_users.js:208
+msgid "s email server"
```

Comment:
This is something to watch out for. Apparently, django's translation string extraction doesn't work for backtick interpolation strings in JS: https://github.com/dimagi/commcare-hq/blob/536a962/corehq/apps/users/static/users/js/web_users.js#L208-L210.